### PR TITLE
Feature: Implement scaffold of components

### PR DESCRIPTION
**Pull request by saszaCodes**
#### What does this PR do?
Implements agreed upon design in React, mirroring the design on Trello and without any functionalities

#### Description of Task to be completed?
Implementing scaffold of components

#### How should this be manually tested?
Run script `npm start` in project directory to see the implemented design

#### Questions:
Should I have merged into the brach the changes that were commited to main while I was working on the branch, resolved potential conflicts and only then created the PR?

### CODE REVIEW

**Comment by Avuidrauxs** [comment]
You have some merge conflicts @tepeszCodes

**Comment by saszaCodes** [comment]
@Avuidrauxs ok, I resolved them

File: package.json [hunk 1]
```diff
@@ -20,8 +22,6 @@
     "lint": "./node_modules/.bin/eslint .",
     "deploy": "node ./scripts/deploy.js"
   },
-  "config": {
-    "branch": "chore/deploy-to-heroku"
   },
```
Comment (Avuidrauxs) [inline package.json]: ### Critical
- 2 closing brackets, breaking the build

File: src/Components/Clock.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating a timer clock
+ * @returns {Node} representing the component
+ */
+function Clock() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto my-0">
+      <p className="text-center fw-bold my-0" style={{ fontSize: '15vw' }}>23:12</p>
```
Comment (Avuidrauxs) [inline src/Components/Clock.js]: ### Nitpick

let's have a variable called `clockStyle` and use it to store our styles rather to be hardcoding styles right in there.

```javascript
const clockStyle = { fontSize: '15vw' }

 <p className="text-center fw-bold my-0" style={clockStyle}>23:12</p>

```

File: src/Components/ClockComment.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating the text to be displayed above the clock
+ * @returns {Node} representing component
+ */
+function ClockComment() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto">
+      <p className="fs-5 fst-italic text-center">More than halfway there!</p>
```
Comment (Avuidrauxs) [inline src/Components/ClockComment.js]: ### Nitpick

- Let's have this to have the string content to be more dynamic so let's make this get a text prop


```suggestion
      <p className="fs-5 fst-italic text-center">{props.text}</p>
```

**Review by Avuidrauxs — CHANGES_REQUESTED** [review]
Great work!! Especially with the `CustomButton` component. Just a few changes and then we should be good to go.

**Review by saszaCodes — COMMENTED** [review]
@Avuidrauxs okay, I implemented / approved all the suggested changes

File: src/Components/ClockComment.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating the text to be displayed above the clock
+ * @returns {Node} representing component
+ */
+function ClockComment() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto">
+      <p className="fs-5 fst-italic text-center">{props.text}</p>
```
Comment (Avuidrauxs) [inline src/Components/ClockComment.js:10]: @tepeszCodes , where is the `props` parameter coming from?

File: src/App.js [hunk 5]
```diff
@@ -1,27 +1,42 @@
...
-    </div>
+    <Col xs={11} md={6} className="m-auto my-4 my-xl-5">
+      <Row>
+        <Col xs={12} md={10} className="m-auto ms-md-0">
+          <InputBar />
+        </Col>
+      </Row>
+      <Row className="mt-4">
+        <Col xs={12} md={10}>
+          <ClockComment />
```
Comment (Avuidrauxs) [inline src/App.js]: @tepeszCodes , doesn't this require props?

File: src/Components/ClockComment.js (hunk 4 above)
Comment (saszaCodes) [reply src/Components/ClockComment.js:10]: My bad, fixed it

File: src/App.js (hunk 5 above)
Comment (saszaCodes) [reply src/App.js]: Of course it does, I overlooked it, fixed it now

**Review by saszaCodes — COMMENTED** [review]
@Avuidrauxs fixed what I overlooked earlier - added prop `text` to the `<ClockComment />` component in App.js and passed it down in ClockComment.js

**Review by Avuidrauxs — APPROVED** [review]
LGTM!

**Pull request merged**